langchain/src/callbacks/utils.ts

```diff
@@ -0,0 +1,49 @@
+import * as process from "process";
+import { LangChainTracer } from "./tracers.js";
+import { CallbackManager, ConsoleCallbackHandler } from "./base.js";
+
+export class SingletonCallbackManager extends CallbackManager {
```

Comment:
I made a `SingleTonCallbackManager` since there should be ideally one instance of `LangChainTracer` (since it is stateful, it has a stack). @nfcampos -- I'm not knowledgable about typescript to know if this will breakdown with concurrency. I know that, if you have multiple async coroutines in python, sharing a tracer is bad news since there could be collisions when the trace is being collected (everything runs on the same thread).